prowler/providers/common/common.py

```diff
@@ -1,7 +1,20 @@
 from importlib import import_module
+from typing import Any
 
 providers_prowler_lib_path = "prowler.providers"
 
+# SHARED PROVIDER OBJECT ACROSS ALL PROWLER CODE
+global_provider = None
+
+
+def set_provider(provider, arguments) -> Any:
+    provider_class_name = f"{provider.capitalize()}Provider"
+    import_module_path = f"prowler.providers.{provider}.azure_provider_testing"
```

Comment:
Watch out, the module path says `azure_provider_testing`, is it expected?

Reply:
Yes, it is a way to make the new implementation cohabit with the old one until we know everything works